lib/metadata_gen.js

```diff
@@ -32,26 +32,21 @@ class MetadataGenerator {
     const fixes = parser.getFixes();
     const refs = parser.getRefs();
 
-    const output = {
-      prUrl, reviewedBy, fixes, refs
-    };
-
     let meta = [
-      `PR-URL: ${output.prUrl}`
+      `PR-URL: ${prUrl}`,
+      ...fixes.map((fix) => `Fixes: ${fix}`),
+      ...refs.map((ref) => `Refs: ${ref}`),
+      ...reviewedBy.map((r) => `Reviewed-By: ${r.reviewer.getContact()}`),
+      '' // creates final EOL
```

Comment:
Same here.

Il love the use of the spread operator, but the last line puzzles me. Isn't there another way to create the final EOL without this line and comment?

Reply:
Just imagine that `.join(str)` puts `str` between all elements in the array so [1, 2, 3] becomes `1str2str3`. When you instead make it [1, 2, 3, ''] it'll instead do `1str2str3str` since there's now an extra empty element.

Reply:
Yup, got that, but since you have to add a comment to make it explicit, I think there must be a more simple and more readable to do it.

I don't like having to rely on comments to understand what code does, I'd prefer you commented why you had to do this, what's the gain ^^

Reply:
I just didn't think that `meta.join(EOL)${EOL}` is any clearer, I would probably need a comment too to make sure no one removes the final EOL.

Reply:
That's a fair point :D
Thanks!